chevah/keycert/ssh.py

```diff
@@ -237,9 +243,14 @@ def __repr__(self):
         """
         Return a pretty representation of this object.
         """
+        type = self.type()
+        if _PY3 and isinstance(type, bytes):
+            type = type.decode('ascii')
+        elif not _PY3 and isinstance(type, unicode):
+            type = type.encode('ascii')
```

Comment:
this is very ugly... I would argue that we should fix self.type() so that it will always return the same data type and that is bytes.